.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - id: set-matrix
+        run: |
+          JSON=$(cat ./composer.json)
+          echo "::set-output name=${JSON//'%'/'%25'}"
+
+  # The actual TER upload job
+  ter-release:
+    needs: set-matrix
+    name: "TYPO3 TER release"
+
+    if: startsWith(github.ref, 'refs/tags/')
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-versions: [ 7.4 ]
```

Comment:
We're hardcoding the the PHP version in line 74. So there is no need to have the PHP version in a matrix.